docs/Security-Compliance/boundary-diagram.md

```diff
@@ -18,17 +18,17 @@ Developers will deploy new code through GitHub, initiating the continuous integr
 
 ### User access
 
-All web users will log into the system through their web browsers. 
+All web users will log into the system through their web browsers. An [Nginx server](../../tdrs-frontend/nginx/README.md) sits in front of each frontend application to serve as a reverse proxy to block traffic to TDP from outside the US/US territories. For traffic *within* the US/US territories:
```

Comment:
The main reason of having that Nginx is to serve the frontend app, so I suggest changing the wording to:

"All web users will log into the system through their web browsers. An [Nginx server](../../tdrs-frontend/nginx/README.md) sits in front of each frontend application to serve the frontend application as well as acting as a reverse proxy which blocks traffic to TDP from outside the US/US territories. For traffic *within* the US/US territories:"